src/Uno.UI/UI/Xaml/Controls/Primitives/SelectorItem.cs

```diff
@@ -170,6 +170,12 @@ private void UpdateCommonStates(ManipulationUpdateKind manipulationUpdate = Mani
 			{
 				// We delay the beginning of a manipulation to avoid flickers, but not for "exact" devices which has hover states
 				// (i.e. mouse and pen when not on iOS)
```

Comment:
About pen: I've confirmed on WinUI that we can scroll a `ListView` using the pen just like a finger This means that we want to apply the same delay on the over state to improve performance of the scrolling.

You should update the comment to reflect it :)

Reply:
```suggestion
				// (i.e. mouse when not on iOS)
```